.gitlab-ci/stages/deploy.yml

```diff
@@ -99,9 +99,11 @@ Deploy @cdtn/api (dev):
       --set deployment.resources.requests.cpu=.3m
 
 Deploy @cdtn/api (prod):
-  <<: *deploy_api_stage
-  <<: *prod_stage
-
+  extends:
+    - .deploy_api_stage
+    - .prod_stage
+  variables:
+    ENABLE_APM: "true"
```

Comment:
you dont need to use ` <<: *deploy_stage_variables` here too ?

Reply:
`deploy_stage_variable` are added to `deploy_api_stage` 
cf https://github.com/SocialGouv/code-du-travail-numerique/pull/2532/files/91c03a0f6488f87efa1cc29f02293dfc0ed5abe2#diff-6cc34a354a467dfca696c0f98c629bfaR76-R80